### DISCUSSION TITLE
I'm contemplating about using a mergefs directory created using multiple disk, to host and run a postgresql instance on it. Do you think this is a good or a bad idea?

### DISCUSSION THREAD

**Discussion by ghevge** [trapexit/mergerfs#1288] [General]
I'm contemplating about using a mergerfs (with snapRAID) directory created using multiple disks (nvme or ssd), to host and run a huge (2TB+)  postgresql DB instance on it.
 Do you think this is a good or a bad idea?  Has anyone did this before?

The appealing part of using mergerfs is the extendibility, as the DB will grow significantly. 

Thanks

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs?tab=readme-ov-file#what-should-mergerfs-not-be-used-for

It is unlikely you'd have any decent performance. I'm also not sure exactly how postgres stores its data and if it is in single large files then mergerfs would do you no good. mergerfs does not give you contiguous storage.
